.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
Question on this:
> I want to make sure that if someone places a testlet in the directory on the server, that the server/assets.go can pick that up and distribute it automatically.

Is this something you want added in the PR? As far as I can see the existing logic in master only instructs the agents to download testlets that are in the assets package.